plot_spectra.py

```diff
@@ -0,0 +1,70 @@
+from collections import deque
+import ast
+from globalvalues import DEFAULT_DATALOG_D3S
+import numpy as np
+import os
+import matplotlib.pyplot as plt
+
+queue = deque('')
+
+def grab_data(path=DEFAULT_DATALOG_D3S):
+    """
+    Takes data from datalog and places it in a queue. Rebin data here.
+    """
+    if os.path.isfile(path):
+        with open(path, 'r') as f:
+            data = f.read()
+        data = ast.literal_eval(data)
+        for i in data:
+            queue.append(rebin(np.array(i)))
+
+def sum_data(data):
+   """
+   Sums up the data in the queue
+   """
+   total = data.popleft()
+   i = 1
+   while i < len(data):
+       total += data.popleft()
+       i+=1
+   return total
+
+def plot_data(data):
+    """
+    Plots data
+    """
+    plt.xlabel('Channel')
+    plt.ylabel('Counts')
+    x = np.linspace(0, len(data), len(data))
+    plt.plot(x, data)
```

Comment:
Add the keyword argument, `drawstyle='steps-mid'` to give a stairstep-type plot. It's a little more precise in visually representing the data.